DependencyInjection/GraphQLiteExtension.php

```diff
@@ -72,7 +72,7 @@ function($namespace): string {
         $container->setParameter('graphqlite.namespace.types', $namespaceType);
         $container->setParameter('graphqlite.security.enable_login', $enableLogin);
         $container->setParameter('graphqlite.security.enable_me', $enableMe);
-        $container->setParameter('graphqlite.security.introspection', $config['security']['introspection'] ?? true);
+        $container->setParameter('graphqlite.security.disableIntrospection', !($config['security']['introspection'] ?? true));
```

Comment:
Reason for inverting the logic here is because i had to pass this argument to the service `DisableIntrospection` which now takes `$enabled` as argument which needs to be `true` for disabling introspection